lib/ledenet/api.rb

```diff
@@ -14,56 +14,112 @@ def initialize(device_address, options = {})
 
     def on
       send_bytes_action(0x71, 0x23, 0x0F, 0xA3)
-
       true
     end
 
     def off
       send_bytes_action(0x71, 0x24 ,0x0F, 0xA4)
-
       true
     end
 
     def on?
-      (status.bytes[13] & 0x01) == 1
+      # Bulbs respond with "35" or "36" (HEX: 23 or 24)
+      # for on and off respectively
+      power_state = status[2].unpack('C').to_s.delete('[]')
+      if  power_state == "35"
```

Comment:
Think you can simplify this to `status[2].unpack('C').to_s.delete('[]') == '35'`

Would also be nice to add a method that does `unpack('C').to_s.delete('[]')` in a method and sorta explain what it does since it's getting used in a bunch of places.